template/ios/HelloWorld.xcodeproj/project.pbxproj

```diff
@@ -557,7 +556,7 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.REACT.HelloWorld-tvOS";
+				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.example.HelloWorld-tvOS";
```

Comment:
These `PRODUCT_BUNDLE_IDENTIFIER ` changes are to bring the tvOS products in line with the iOS one.